deployment/helm/tensorwave70b/scalarlm/templates/api_deployment.yaml

```diff
@@ -44,10 +44,10 @@ spec:
       volumes:
         - name: scalarlm-jobs
           persistentVolumeClaim:
-            claimName: scalarlm-70b-jobs
+            claimName: scalarlm-jobs
         - name: scalarlm-cache
           persistentVolumeClaim:
-            claimName: scalarlm-70b-cache
+            claimName: scalarlm-cache
```

Comment:
minor clean up / refactor since we are using distinct namespaces based on model sized in deployments.